src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneExecutor.java

```diff
@@ -72,6 +72,7 @@ public void runLoad(BulkLoadFile bulkLoadFile, Boolean cleanUp) {
 			trackHistory(history, bulkLoadFile);
 
 		} catch (Exception e) {
+			failLoad(bulkLoadFile, e.getMessage());
```

Comment:
`e.getMessage()` might not be the best message to put as the error message. There are a few others that could be added. There are several others:
```
			e.getLocalizedMessage();
			e.getCause().getMessage();
```
These being added to the error message might be more informative.